nca/NetworkConfig/PoliciesFinder.py

```diff
@@ -25,6 +25,9 @@ def __init__(self):
         self.policies_container = PoliciesContainer()
         self._parse_queue = deque()
         self.peer_container = None
+        self.missing_gw_peers = False
+        self.found_ingress_control_policy = False
```

Comment:
consider renaming `missing_gw_peers` to `missing_istio_gw_peers`, and `found_ingress_control_policy` to `found_k8s_ingress_policy`

Reply:
done